File: Go/uri_1101.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var m, n = 1, 1
	for {
		s := 0
		_, _ = fmt.Scanf("%d%d", &m, &n)
		if m <= 0 || n <= 0 {
			break
		} else {
			maxx := int(math.Max(float64(m), float64(n)))
			minn := int(math.Min(float64(m), float64(n)))
			for i := minn; i <= maxx; i++ {
				s += i
				fmt.Printf("%d ", i)
			}
			fmt.Printf("Sum=%d\n", s)
		}
	}

}

File: Python/uri_1101.py
on = True
soma = 0
while on:
    m, n = map(int, input().split(" "))
    soma = 0
    if (m <= 0) or (n <= 0):
        break
    maxx = max(m, n)
    minn = min(m, n)
    for i in range(minn, maxx + 1):
        print("%d " % i, end="")
        soma += i
    print("Sum=%d" % soma)
